google-cloud-spanner/src/main/resources/com/google/cloud/spanner/connection/ClientSideStatements.json

```diff
@@ -158,6 +158,33 @@
 		"method": "statementShowDelayTransactionStartUntilFirstWrite",
 		"exampleStatements": ["show variable delay_transaction_start_until_first_write"]
 	},
+	{
+		"name": "PARTITION <sql>",
+		"executorName": "ClientSideStatementPartitionExecutor",
+		"resultType": "RESULT_SET",
+		"statementType": "PARTITION",
+		"regex": "(?is)\\A\\s*partition(\\s+|\\()(.*)\\z",
+		"method": "statementPartition",
+		"exampleStatements": []
```

Comment:
Referring back to a previous comment of mine where I asked for a few examples, can we add a few examples here which can be used to understand the `ClientSideStatementPartitionExecutor` code better?